htmlgenerator/base.py

```diff
@@ -270,7 +270,7 @@ def treewalk(
         typing.Callable[[BaseElement, typing.Tuple[BaseElement, ...]], bool]
     ],
     apply: typing.Optional[
-        typing.Callable[[BaseElement, int, BaseElement], None]
+        typing.Callable[[typing.List, int, BaseElement], None]
```

Comment:
I think this should stay BaseElement, but the method parameter ```element``` should be changed to ```element```

Reply:
Sorry, I mean: changed to ```BaseElement```